src/main/java/io/javalin/community/ssl/TrustConfig.java

```diff
@@ -0,0 +1,123 @@
+package io.javalin.community.ssl;
+
+import nl.altindag.ssl.util.CertificateUtils;
+import nl.altindag.ssl.util.KeyStoreUtils;
+
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.cert.Certificate;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Configuration for the trust store, used to verify the identity of the clients.
+ * Using this configuration, the server will only accept connections from clients that are trusted.
+ * If no trust store is configured, the server will accept any client.
+ */
+
+public class TrustConfig {
+
+
+    /**
+     * List of certificates to be trusted, can be loaded using the helper methods or directly.
+     * This list is complementary to the keys
+     */
+    public List<Certificate> certificates = new ArrayList<>();
```

Comment:
Wouldn't it be better to make this private?

Reply:
No, it is an advanced configuration, I don't really care if the user loads the certificates their own way and replaces the array, it's only used while loading them into your plugin. It would only be an issue if you don't make a copy of the list yourself and just hold the reference to the users, do you?

Reply:
Aha so it will be only used for the initial configuration and if the array changes during the lifecycle it won't effect the state of the server. 

It is just a common practise to encapsulate class fields and make it immutable to prevent changing the state, so that is why I suggested it, but it seems like it won't harm in your use case.

Reply:
Oh I know, thanks for the review though! It's mostly the style of Javalin's configs.